libvalhalla.pc.in

```diff
@@ -1,10 +1,10 @@
-prefix=@prefix@
-exec_prefix=@exec_prefix@
-libdir=${exec_prefix}/@libdir@
-includedir=${prefix}/@includedir@
+prefix=@CMAKE_INSTALL_PREFIX@
+exec_prefix=${prefix}
+libdir=${exec_prefix}/@CMAKE_INSTALL_LIBDIR@
+includedir=${prefix}/@CMAKE_INSTALL_INCLUDEDIR@
 
 Name: libvalhalla
 Description: valhalla c++ library
 Version: @VERSION@
-Libs: -L${libdir} -lvalhalla @deplibs@
+Libs: -L${libdir} -lvalhalla -lprotobuf-lite -pthread -lcurl -lz
```

Comment:
this cant possibly be the full list if `DATA_TOOLS` is enabled (which it most certainly should be in the default case). have you tried the standard print out all the cmake variables trick to see if one of the variables has all the dependencies in it :smile: every time i want to figure out somethign like this i fall back to this: https://stackoverflow.com/questions/9298278/cmake-print-out-all-accessible-variables-in-a-script

Reply:
It isn't no, I currently use valhalla without `DATA_TOOLS` :sweat_smile: 

I have yet to figure out how to do this properly sadly... Will give what you linked a shot

Reply:
Well I printed out everything but still don't really see what I need...
[cmake.log](https://github.com/valhalla/valhalla/files/6128688/cmake.log)